Can't install with engine-strict

Runnning `npm install @storybook/react --engine-strict` (or with `engine-strict` in .npmrc) with npm 5 fails with

```
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for podda@1.2.2: wanted: {"npm":"^3.0.0"} (current: {"node":"8.9.1","npm":"5.6.0"})
npm ERR! notsup Not compatible with your version of node/npm: podda@1.2.2
npm ERR! notsup Not compatible with your version of node/npm: podda@1.2.2
npm ERR! notsup Required: {"npm":"^3.0.0"}
npm ERR! notsup Actual:   {"npm":"5.6.0","node":"8.9.1"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/felix/.npm/_logs/2018-02-19T22_21_14_142Z-debug.log
```
Afaik yarn has this on by default too.

Dependency tree:

```
sourcegraph@0.0.1 /Users/felix/go/src/sourcegraph.com/sourcegraph/sourcegraph/web
└─┬ @storybook/react@3.3.13
  └─┬ @storybook/ui@3.3.13
    └── podda@1.2.2
```

Tracked in https://github.com/arunoda/podda/issues/6, but obviously severely impacting storybook.
PR has been open since August: https://github.com/arunoda/podda/pull/5

--- a/lib/ui/package.json
+++ b/lib/ui/package.json
@@ -28,7 +28,7 @@
     "lodash.debounce": "^4.0.8",
     "lodash.pick": "^4.4.0",
     "lodash.sortby": "^4.7.0",
-    "podda": "^1.2.2",
+    "@storybook/podda": "^1.2.3",
     "prop-types": "^15.6.0",
     "qs": "^6.5.1",
     "react-fuzzy": "^0.5.2",

--- a/lib/ui/src/index.js
+++ b/lib/ui/src/index.js
@@ -1,5 +1,5 @@
 import { createApp } from '@storybook/mantra-core';
-import Podda from 'podda';
+import Podda from '@storybook/podda';
 
 import buildContext from './context';
 import shortcutsModule from './modules/shortcuts';

--- a/lib/ui/src/modules/ui/libs/gen_podda_loader.test.js
+++ b/lib/ui/src/modules/ui/libs/gen_podda_loader.test.js
@@ -1,4 +1,4 @@
-import Podda from 'podda';
+import Podda from '@storybook/podda';
 import genPoddaLoader from './gen_podda_loader';
 
 describe('manager.ui.libs.gen_podda_loader', () => {

